server/templates/student/course/_assigntable.html

```diff
@@ -9,7 +9,7 @@
           {% if scores %}<th class="col-scores">Scores</th> {% endif %}
           <th class="col-status">Status</th>
           <th class="col-due-date">Due Date</th>
-          <th class="col-group">Group</th>
+          <th class="col-group hidden-md">Group</th>
```

Comment:
Make it a little less crowded on the md view (992px to 1199px).

Result (1024px):

![screen shot 2016-08-24 at 20 22 08](https://cloud.githubusercontent.com/assets/1915715/17955757/558b7154-6a39-11e6-9ae8-38fcde3acbd9.png)

1280px still looks fine in my opinion.

![screen shot 2016-08-24 at 20 25 11](https://cloud.githubusercontent.com/assets/1915715/17955761/62e62614-6a39-11e6-983d-af45cf42874b.png)